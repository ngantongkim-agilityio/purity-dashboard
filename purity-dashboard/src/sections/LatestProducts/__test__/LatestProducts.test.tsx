import { render } from '@testing-library/react';

// Components
import { LatestProducts } from '..';

jest.mock('@/services/products.ts', () => ({
  fetchLatestProducts: jest.fn()
}));

describe('LatestProducts component', () => {
  it('Should render LatestProducts component with default props', () => {
    const { container } = render(<LatestProducts />);

    expect(container).toMatchSnapshot();
  });
});
