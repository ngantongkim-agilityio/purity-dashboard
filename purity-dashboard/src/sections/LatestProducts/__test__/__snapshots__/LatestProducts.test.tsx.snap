// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestProducts component Should render LatestProducts component with default props 1`] = `<div />`;
